### PR TITLE
Docs: Shortening home page copy

### LINKED ISSUE #273: Shorten up those features on the home page
Home page could use a bit of a sprucing:
* Shorten up the four features listed below the fold
* reduce taglines to just one

### PR DESCRIPTION
**Pull request by rachelnabors**
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Ref https://github.com/sonarwhal/sonarwhal.com/issues/273
Fix #273
Close #273

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonar)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/developer-guide/contributing/pull-requests.html#commitmessages)

### CODE REVIEW

File: src/hexo/themes/sonarwhal/layout/partials/home.hbs [hunk 1]
```diff
@@ -12,53 +12,49 @@
...
                     <div class="layout layout--quarters">
                         <div class="module">
-                            <p class="subtitle--alt subtitle--home uppercase-text">Accessibility, security, interoperability and more!</p>
+                            <p class="subtitle--alt subtitle--home uppercase-text">Don’t burnout&hellip;</p>
                             <p class="feature_highlight">
-                                The web is complex. Knowing everything about web development and not making any mistakes can be challenging.
-                                sonar is meant to help developers regardless of their skill level, from the novice to the
-                                experienced one, to build websites and apps that work for everyone, are accessible, fast, secure, and
-                                take advantage of all the features the web has to offer.
+                                &hellip;trying to master all the things: sonar will help you ensure your code is accessible, fast, secure,
```
Comment (molant) [inline src/hexo/themes/sonarwhal/layout/partials/home.hbs]: I think people could interpret this like "if I pass all the rules in sonar, my website is going to be fully accessible, fast, secure, etc." which is not the case. We will help you in those areas, but there are things we are not checking that could help in these and other areas.

What about something like:

> sonar will help you with your site's accessibility, speed, security, and more

File: src/hexo/themes/sonarwhal/layout/partials/home.hbs [hunk 2]
```diff
@@ -12,53 +12,49 @@
...
-                    <p class="title">a linting tool for the web</p>
+                    <p class="title">build better web sites&mdash;lint the web forward</p>
                     <div class="actions">
                         <a class="button button-cta button--red" href="/docs/user-guide/">Getting Started</a>
                         <a class="button button-cta button--red" href="/scanner/">Try it online</a>
                     </div>
                 </section>
-                <section class="home-container--section home-container__features">
-                    <h2 class="subheading subheading--home">Build better websites with sonar</h2>
+                <div class="home-container--section home-container__features">
```
Comment (molant) [inline src/hexo/themes/sonarwhal/layout/partials/home.hbs]: Why did you changed `section` to `div`?

**Comment by rachelnabors** [comment]
Because section elements require their first child to be a heading to
validate; to pull the text up, I had removed that section's heading. It
didn't provide more impact than the content that followed, and the heading
at the top could be retooled to say the same thing, but with more punch.

**Comment by ststimac** [comment]
I would prefer we keep some sort of heading where you removed the one @rachelnabors. The home page is chunked out into sections on purpose and the headings help to group those chunks of content and divide the page up. If you can come up with something more impactful for that heading that would be great.

File: src/hexo/themes/sonarwhal/layout/partials/home.hbs (hunk 1 above)
Comment (molant) [reply src/hexo/themes/sonarwhal/layout/partials/home.hbs]: @rachelnabors any thoughts about my comment above?

**Comment by molant** [comment]
@rachelnabors looks like there are conflicts with this PR. Can you please fix those when yo take care of the comments?
Thanks!

File: src/hexo/themes/sonarwhal/layout/partials/home.hbs (hunk 1 above)
Comment (rachelnabors) [reply src/hexo/themes/sonarwhal/layout/partials/home.hbs]: I have no concerns!

**Comment by rachelnabors** [comment]
@ststimac I see your point, Stephanie. Are the headings there more for aesthetics than for content?

**Pull request merged**